Sharpmake/Project.cs

```diff
@@ -173,6 +173,8 @@ public int SourceFilesFiltersCount
 
         public Strings XResourcesResw = new Strings();
 
+        public bool EnableDefaultItems { get; set; } = false;                           // Enable or disable the property [EnableDefaultItems] in NetCore Project Schema
```

Comment:
Please use xmldoc format to document the setting

Reply:
done